Jenkinsfile_CNP

```diff
@@ -18,6 +18,7 @@ def secrets = [
   'ccd-${env}': [
     secret('ccd-befta-caseworker-2-solicitor-2-pwd', 'CCD_BEFTA_CASEWORKER_2_SOLICITOR_2_PWD'),
     secret('ccd-api-gateway-oauth2-client-secret', 'CCD_API_GATEWAY_OAUTH2_CLIENT_SECRET'),
+    secret('ts-service-translation-users-pwd', 'TS-SERVICE-TRANSLATION-USERS-PWD'),
```

Comment:
Should this use underscores (in both secret name and the env var) to be consistent with other secrets?

Reply:
Looks like secrets keys should probably remain in `kebab-case` but agree env vars should change to `snake_case` for consistency with others.

Reply:
done 😊